{
  "comments": [
    {
      "key": {
        "uuid": "9a437d4f_e3ee238b",
        "filename": "recovery/recovery_updater.cpp",
        "patchSetId": 3
      },
      "lineNbr": 136,
      "author": {
        "id": 1063
      },
      "writtenOn": "2017-01-27T14:17:31Z",
      "side": 1,
      "message": "This call can fail and return -1 ... should error out in that case.",
      "revId": "126f591786c4e3edbd20f802bbcd0dcd6994a641",
      "serverId": "1ec6b3db-b2c1-4fa4-84a2-4c7efe89ba71",
      "unresolved": false
    }
  ]
}